### DISCUSSION TITLE
ComboBox 鼠标左键点击后没有任何反应，需要鼠标右键进行点击

### DISCUSSION THREAD

**Discussion by xingwa** [AvaloniaUI/Avalonia#15249] [Q&A]
### Describe the bug

ComboBox 鼠标左键点击后没有任何反应，需要鼠标右键进行点击才能正常选择。

### To Reproduce

ComboBox 鼠标左键点击后没有任何反应，需要鼠标右键进行点击才能正常选择。

### Avalonia version

11.0.10

**Comment by rabbitism** [comment]
什么操作系统？

**Comment by xingwa** [comment]
win 10

**Comment by xingwa** [comment]
![微信图片_20240405205335](https://github.com/AvaloniaUI/Avalonia/assets/15813393/dfc12cb6-40f5-4827-b745-06f097fd4072)

![Uploading 微信图片_20240405205504.png…]()

**Comment by timunie** [comment]
made it a Q&A until we understand the issue.

**Comment by 1324278444** [comment]
我也碰到这个问题了，在预览界面可以使用鼠标左键打开，但是运行起来就不行，只能用鼠标右键打开，你那儿解决了吗

**Reply by 1324278444** [reply]
我知道我的为什么打不开，因为我给window设置了BeginMoveDrag，去掉这个就正常了
